schemas/xchem_db/init.sql

```diff
@@ -1,4 +1,8 @@
 --
+-- Add field comment to reviewresponses
```

Comment:
This ALTER TABLE should be placed after the corresponding CREATE TABLE, or perhaps better: Modify the CREATE TABLE so you don't have to do the ALTER TABLE at all.

Reply:
Okay I think I have done it correctly...